### PR TITLE
WIP Add requirements.txt and scipt to build env.yml

### PR DESCRIPTION
**Pull request by drvinceknight**
This relates to #299.

Opening a PR to check that it works. Will subsequently trim the
install.sh which will have things that are not needed.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+numpy==1.11
+scipy==0.19
+scikit-learn==0.18
+matplotlib==1.5
+scikit-image==0.12
+sympy==1.0
+statsmodels=0.6
```
Comment (gertingold) [inline requirements.txt]: In the `statsmodels` line there should be two equal signs.

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Do not manually modify this file.
```
Comment (GaelVaroquaux) [inline environment.yml]: Be more explicit: say that it's generated, and say what generates it.

**Comment by GaelVaroquaux** [comment]
The approach is the good one.

We should probably rename the "continuous_integration" director to "build_utils" (or "build_tools") and move the "requirements-to-environment.py" in there.

Also: minor: I would renamed "requirements-to-environment.py" to "requirements_to_environment.py". The reason being that dashes in Python file names makes them not importable, and hence it is to be avoided.

**Comment by GaelVaroquaux** [comment]
Merging. Great job! Thanks

**Pull request merged**